pipeline-scripts/create-database.sql

```diff
@@ -952,4 +953,25 @@ DO
 DELETE FROM auth_success WHERE Expiry < UTC_TIMESTAMP();
 //
 
+-- --------
+-- --------
+-- Stored Procedures
+-- --------
+-- --------
+
+CREATE PROCEDURE UpdateMarketSummaryLiquidityPoolCount(IN marketId BIGINT UNSIGNED, IN blockHeight BIGINT UNSIGNED)
+    BEGIN
+        DECLARE _marketId BIGINT UNSIGNED DEFAULT marketId; # clear up ambiguity
```

Comment:
DataGrip is showing that `WHERE ms.MarketId = marketId` is always true, as the column name matches the variable name. Redeclared the param with a different name, although thinking we instead use a `_` prefix for all variable names in stored procs.